docs/development/getting-started-remotely.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
You could link the new document from the documentation index (the root README.md).